demo/java_quickstart/demo/pom.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
```

Comment:
does docker-compose test covered ?

Reply:
Next pr, it's a big change, we'll add 3 quickstart demo and some use cases.